buildSrc/DevBuild.js

```diff
@@ -181,6 +181,12 @@ globalThis.buildOptions.sqliteNativePath = "./better-sqlite3.node";`,
 	await runStep("Desktop: assets", async () => {
 		const desktopIconsPath = "./resources/desktop-icons"
 		await fs.copy(desktopIconsPath, "./build/desktop/resources/icons", { overwrite: true })
+		if (stage === "local" || stage === "host") {
```

Comment:
I think dev build should do it for all stages

Reply:
I'm a bit confused, if all stages are dev builds, than how do you build non-dev builds?

Reply:
All builds from DevBuild.js are dev. The production one uses buildSrc/webapp.js

Reply:
prod build is `desktop.js`

Reply:
Ok fixed.